### PR TITLE
CLOUD-323 Kubectl error tolerance

### PR DESCRIPTION
**Pull request by cap1984**
In order to make tests more stable to kubectl errors
retry mechanism has been implemented

### CODE REVIEW

File: e2e-tests/scheduled-backup/run [hunk 1]
```diff
@@ -68,18 +68,18 @@ start_minio() {
         --set environment.MINIO_REGION=us-east-1 \
         --set environment.MINIO_HTTP_TRACE=/tmp/trace.log \
         stable/minio
-    MINIO_POD=$(kubectl get pods --selector=release=minio-service -o 'jsonpath={.items[].metadata.name}')
+    MINIO_POD=$(kubectl_bin get pods --selector=release=minio-service -o 'jsonpath={.items[].metadata.name}')
```
Comment (delgod) [inline e2e-tests/scheduled-backup/run]: function run command few times, so function output can have artifacts in output from all runs (not only the last)

File: e2e-tests/functions [hunk 2]
```diff
@@ -425,7 +429,26 @@ spinup_pxc() {
...
+        kubectl "$@" 2>${error_container}
+        exit_status="$?"
+
+        if [[ ${exit_status} != 0 ]]; then
+            sleep "$((timeout * i))"
+        else
+            return ${exit_status}
+        fi
+    done
+    cat ${error_container}
```
Comment (delgod) [inline e2e-tests/functions]: stderr output should go to stderr

File: e2e-tests/scheduled-backup/run (hunk 1 above)
Comment (cap1984) [reply e2e-tests/scheduled-backup/run]: fixed

File: e2e-tests/functions (hunk 2 above)
Comment (cap1984) [reply e2e-tests/functions]: done

**Comment by JNKPercona** [comment]
PXC operator docker - perconalab/percona-xtradb-cluster-operator:PR-224-a971236

**Comment by JNKPercona** [comment]
PXC operator docker - perconalab/percona-xtradb-cluster-operator:PR-224-a971236

**Comment by JNKPercona** [comment]
PXC operator docker - perconalab/percona-xtradb-cluster-operator:PR-224-a971236

**Comment by JNKPercona** [comment]
PXC operator docker - perconalab/percona-xtradb-cluster-operator:PR-224-6055cfd

**Comment by JNKPercona** [comment]
PXC operator docker - perconalab/percona-xtradb-cluster-operator:PR-224-778230d

**Comment by JNKPercona** [comment]
PXC operator docker - perconalab/percona-xtradb-cluster-operator:PR-224-0737a84

**Comment by JNKPercona** [comment]
PXC operator docker - perconalab/percona-xtradb-cluster-operator:PR-224-9f3185e

**Comment by JNKPercona** [comment]
PXC operator docker - perconalab/percona-xtradb-cluster-operator:PR-224-9f3185e

File: e2e-tests/functions (hunk 2 above)
Comment (delgod) [reply e2e-tests/functions]:
```suggestion
    cat ${error_container} 1>&2
```

File: e2e-tests/functions [hunk 3]
```diff
@@ -207,21 +202,26 @@ run_mysql() {
 
     client_pod=$(get_client_pod)
     wait_pod $client_pod 1>&2
-    kubectl exec $client_pod -- \
-        bash -c "printf '$command\n' | mysql -sN $uri" 2>&1 \
+
+    set +o xtrace
```
Comment (delgod) [inline e2e-tests/functions]: why is it needed?

Comment (delgod) [reply e2e-tests/functions]: sorry, please ignore

**Comment by JNKPercona** [comment]
PXC operator docker - perconalab/percona-xtradb-cluster-operator:PR-224-d35b167

**Comment by JNKPercona** [comment]
PXC operator docker - perconalab/percona-xtradb-cluster-operator:PR-224-c8bb73d

**Pull request merged**